pinot-plugins/pinot-input-format/pinot-parquet/src/test/java/org/apache/pinot/plugin/inputformat/parquet/ParquetNativeRecordReaderFullTest.java

```diff
@@ -123,19 +123,22 @@ protected void testReadDataSet2()
     //testParquetFile("test-data2/uniform_encryption.parquet.encrypted");
   }
 
-  protected void testParquetFile(String filePath)
+protected void testParquetFile(String filePath)
       throws Exception {
     File dataFile = new File(URLDecoder.decode(getClass().getClassLoader().getResource(filePath).getFile(), "UTF-8"));
     ParquetNativeRecordReader recordReader = new ParquetNativeRecordReader();
     recordReader.init(dataFile, null, null);
+    int counter = 0;
```

Comment:
Would it be more explicit if we count `numDocsForFirstPass` and `numDocsForSecondPass` and compare them? Can you also use try-with-resource to close the record reader (not introduced in this PR)